src/queues/sqs/notification/mod.rs

```diff
@@ -139,10 +140,10 @@ pub struct Mail {
     sending_account_id: Option<String>,
     destination: Option<Vec<String>>,
     #[serde(rename = "headersTruncated")]
-    headers_truncated: Option<String>,
+    headers_truncated: Option<bool>,
     headers: Option<Vec<Header>>,
     #[serde(rename = "commonHeaders")]
-    common_headers: Option<Vec<Header>>,
+    common_headers: Option<HashMap<String, HeaderValue>>,
```

Comment:
Should the `Header` circa line 160 be deleted too?

Reply:
Oh, wait, no we shouldn't, it's still used for the `headers` property on the line above.

(apologies for my poor reading comprehension! 😄)

Reply:
I thought about writing a deserializing function for the `common_headers` so that we can keep it as a `Option<Vec<Header>>`, it makes sense to me that both headers are the same type... But then I thought that maybe we might want the struct to mimic exactly the message we get. What do you think, @philbooth ?

Reply:
I reckon keep it like you've got here, it's probably less confusing if we match the SES format as closely as possible.